pkg/client/migrations/1_v0.17.4.up.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TABLE IF NOT EXISTS cq_fetches
```

Comment:
we can call this table fetches as users will do cloudquery.fetches to access this table `cq` will be redundant here i.e cloudquery.cq_fetches